definitions/divorce/json/CaseEvent/CaseEvent-prod.json

```diff
@@ -154,5 +54,61 @@
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
     "SecurityClassification": "Public",
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "15/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "aosSubmittedUndefended",
+    "Name": "AOS Received (undefended)",
+    "Description": "AOS Received (undefended)",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "AosStarted;AosOverdue;ServiceApplicationNotApproved",
+    "PostConditionState": "AwaitingDecreeNisi",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/aos-received",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/aos-submitted",
+    "RetriesTimeoutURLSubmittedEvent": "120,120",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "15/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "aosSubmittedDefended",
+    "Name": "AOS Submitted (defending)",
+    "Description": "AOS Submitted (defending)",
+    "DisplayOrder": 2,
+    "PreConditionState(s)": "AosStarted;AosOverdue;ServiceApplicationNotApproved",
+    "PostConditionState": "AosSubmittedAwaitingAnswer",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/aos-received",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/aos-submitted",
+    "RetriesTimeoutURLSubmittedEvent": "120,120",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "15/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "aosReceivedNoAdConStarted",
+    "Name": "AOS Received (no ad/con)",
+    "Description": "AOS Received (no admission/consent)",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "AosStarted;AosOverdue;ServiceApplicationNotApproved",
+    "PostConditionState": "AosCompleted",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/aos-received",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/aos-submitted",
+    "RetriesTimeoutURLSubmittedEvent": "120,120",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/12/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "boWelshReview",
+    "Name": "Welsh Translation Review",
+    "Description": "Welsh Translation Review",
+    "DisplayOrder": 2,
+    "PreConditionState(s)": "AosStarted;AosOverdue",
```

Comment:
I think here as "PreConditionState" you can add "ServiceApplicationNotApproved" to solve https://tools.hmcts.net/jira/browse/RPET-688. Or leave the whole event in "CaseEvent.json" so it is solved separately when the PET team works on: https://tools.hmcts.net/jira/browse/RPET-688 (However it might take a long time before they get to that)

Reply:
I will add a comment to RPET-668 with link to your comment to make sure they will address it when time comes.